mainnet/docker-compose.yml

```diff
@@ -52,6 +73,12 @@ volumes:
       type: none
       device: /private/var/lib/bitcoin
       o: bind
+  elements:
+    driver: local
+    driver_opts:
+      type: none
+      device: /private/var/lib/elements
```

Comment:
This path is because of me initially playing on mac os (and it uses `/private/var` instead of `/var`, otherwise docker daemon has problems). However it will be nice to have it platform-dependent and generally configurable. What is the best way of doing that?

Reply:
Yeah i kept it like that for consistency with the other containers but i also wanted to find something more generalized.

I'm not sure what's the best answer because i'm not really a docker expert, probably using volumes?

Reply:
volumes are, but I love to have blockchain navigatable from `ls` command or through file browser with predictable path — and I assume many others too. Anyway, seems like there is no other viable option other than volumes (or generation of compose files with some script and parameters, which I also do not like)

Reply:
Ha, what I have found: [It’s possible to use environment variables in your shell to populate values inside a Compose file](https://docs.docker.com/compose/environment-variables/#substitute-environment-variables-in-compose-files)

and we can use external env files with a set of env constants - and just replace that. Will create an issue for that, let's do that in a separate PR